docker/flux/checks-annotate.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
```

Comment:
Is this vendored in from somewhere? Perhaps we should credit that in the commit message? This will help us as well. If there is a bug in a vendored-in script, we can first look at the upstream repo.

Reply:
If this is not copied from somewhere else, please ignore my posting above.

Reply:
That's a good point.  It is from flux-core.  I'll update the commit message.  I'll also break it up into two commits (one that vendors the file and another that customizes it for this repo).